src/pages/relicans.js

```diff
@@ -0,0 +1,347 @@
+        {
+          name: "twitter",
+          url: "https://twitter.com/ChaelCodes"
+        },
+        {
+          name: "twitch",
+          url: "https://twitch.tv/ChaelCodes",
+        },
+        {
+          name: "linkedin",
+          url: "https://www.linkedin.com/in/rachael-wright-chaelcodes/"
+        }
+      ]
+    }
+  ]
+
+  const teamMemberPanels = teamMembers.map(teamMember => {
+    const socialItems = teamMember.socials.map(socialNetwork => {
+      return (
+        <li>
+            <a href={socialNetwork.url}>
+                <FeatherIcon name={socialNetwork.name} size="1.5rem" />
+            </a>
+        </li>
+      )
+    })
+
+    return (
+      <div className={styles.point}>
+        <img src={teamMember.avatar} className={styles.avatar}/>
+        <h3>{teamMember.name}</h3>
+        <h4>{teamMember.pronouns}</h4>
+        {teamMember.bio}
+          <ul className={styles.socials}>
+              {socialItems}
+          </ul>
+      </div>
+    )
+  })
+
+  return (
+    <>
+      <SEO />
+      <PageLayout type={PageLayout.TYPE.SINGLE_COLUMN}>
+        <PageLayout.Header title="Relicans - The New Relic Developer Relations Team" />
+        <PageLayout.Content>
+          <section className={cx(styles.section, styles.twoColumn)}>
+            <div>
+              <p>
+                As the founder of SaaS application performance management, New Relic of course loves  software. But it’s not just about the software. Or performance. Or infrastructure, observability, Kubernetes, Prometheus and all the other technologies we write about and whose performance we help you understand. We love the people who make software--the developers, SREs, engineers who code and test and deploy and deliver to ensure the best end-user experience possible. Our entire platform is designed to make your lives easier. All of which is why we’re so very proud to announce a developer relations team whose purpose will be:
+              </p>
+              <ul>
+                <li>
+                  Producing educational and inspirational content for anyone who writes code, whether in development, site reliability engineering, or IT operations.
+                </li>
+                <li>
+                  To partner with you as you explore the world of software--not just on our platform but with all the languages, frameworks, and tools you love.
+                </li>
+                <li>
+                  To invite you to hang out with us for gaming, writing code, playing music, just having fun.
+                </li>
+              </ul>
+              
+            </div>
+            <iframe width="560" height="315" src="https://www.youtube.com/embed/OlW3g631ero" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+          </section>
+
+          <h2 style={{ textAlign: 'center', marginBottom: 30, background: '#fff'}}>Meet the Relicans</h2>
```

Comment:
The treatment on this H2 in dark mode seems off and doesn't follow our pattern for dark mode, I'd suggest just following how we toggle font colors from dark to light. In dark mode all headers font colors are changed to color: rgb(232, 234, 234)

<img width="1380" alt="Screen Shot 2021-01-27 at 9 04 25 AM" src="https://user-images.githubusercontent.com/1395158/106019485-f1df7a00-607f-11eb-9beb-0ca4a1eb5f0a.png">